src/App.js

```diff
@@ -1,16 +1,75 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog.js';
+import { useState } from 'react';
```

Comment:
Small nit, since L1 and L5 both import classes/functions from `react`, you can combine them in the same line:
```
import React, { useState } from 'react';
```